pom.xml

```diff
@@ -188,7 +188,7 @@
     <dependency>
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcpkix-jdk15to18</artifactId>
-      <version>1.74</version>
+      <version>1.78</version>
```

Comment:
I see that 1.74 has some vulnerabilities that was fixed in 1.78 - so my suggestion will be to include this change together with this vulnerability fix: https://github.com/folio-org/edge-sip2/pull/165 into release
![image](https://github.com/folio-org/edge-sip2/assets/25097693/d12b9c96-4264-48e9-b73d-41e98b3ba2d8)